domains/Discussions/Controllers/AnswersUpdateController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+
+namespace Domains\Discussions\Controllers;
+
+
+use App\Http\Controllers\Controller;
+use Carbon\Carbon;
+use Domains\Discussions\Models\Answer;
+use Domains\Discussions\Models\Question;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class AnswersUpdateController extends Controller
+{
+    private Answer $answer;
+    private Question $question;
+
+    public function __construct(Question $question, Answer $answer)
+    {
+        $this->question = $question;
+        $this->answer = $answer;
+    }
+
+    public function __invoke(int $questionId, int $answerId, Request $request): Response
+    {
+        $answer = $this->answer->where("question_id", $questionId)->findOrFail($answerId);
```

Comment:
Let's use single quote to make it consistent.
Also the `where(..)` is not needed since we're targeting a single answer with `findOrFail`

Reply:
This will be impacted by the answer to your question on the issue. #33

Reply:
@adevr @josepostiga already commented on this below. So please keep the where and just use the single quote.